core/domain/prod_validation_jobs_one_off.py

```diff
@@ -281,6 +285,9 @@ def _get_custom_validation_functions(cls):
 
         Each validation function should accept only a single arg, which is the
         model instance to validate.
+
+        Returns:
+            Returns the list of custom validation functions to run.
```

Comment:
Return type first then description, check the wiki page for more info.

Reply:
it's returing empty list, so should i just write "list. The list of custom validation functions to run.", confused!!

Reply:
Yeah!

Reply:
done